client/ayon_ftrack/plugins/publish/integrate_hierarchy_ftrack.py

```diff
@@ -273,7 +273,7 @@ def import_to_ftrack(
 
             # Create entity if not exists
             if entity is None:
-                folder_type = entity_data["folder_type"]
+                folder_type = entity_data["folder_type"].capitalize()
```

Comment:
Why `capitalize`?

Reply:
Ok, will investigate a more flexible solution. For context, this is the error:

```
Traceback (most recent call last):
  File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "C:\Users\tokejepsen\ayon-ftrack\client\ayon_ftrack\plugins\publish\integrate_hierarchy_ftrack.py", line 75, in process
  File "C:\Users\tokejepsen\ayon-ftrack\client\ayon_ftrack\plugins\publish\integrate_hierarchy_ftrack.py", line 277, in import_to_ftrack
  File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\ftrack_api\session.py", line 643, in create
    entity = self._create(entity_type, data, reconstructing=reconstructing)
  File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\ftrack_api\session.py", line 652, in _create
    raise ftrack_api.exception.UnrecognisedEntityTypeError(entity_type)
ftrack_api.exception.UnrecognisedEntityTypeError: Entity type "b'sequence'" not recognised.
```

Reply:
```python
ft_project_schema = ft_project["project_schema"]
object_types_by_low_name = {
    object_type["name"].lower(): object_type["name"].replace(" ", "")
    for object_type in ft_project_schema["object_types"]
}
```

You can then use
```python
folder_type_low = entity_data["folder_type"].lower()
entity_type = object_types_by_low_name [folder_type_low]
entity = session.create(entity_type, {
    ...
})
```